core/src/widgets/audio_widgets/knob.rs

```diff
@@ -209,11 +220,11 @@ impl Widget for Knob {
         let move_virtual_slider = |_self: &mut Self, state: &mut State, new_normal: f32| {
```

Comment:
I don't think this should be prefixed with an underscore since it is being used.

Reply:
It won't actually let you use `self` as a parameter name in a closure. I can call it something like `self_ref` or something.